muselog/bootstrap/sitecustomize.py

```diff
@@ -0,0 +1,17 @@
+"""Support application-wide exception hook with muselog integration."""
```

Comment:
Why is this file called `sitecustomize`?

Reply:
This is required by python. It looks for a file with this name in the site path and executes it if it exists.